src/code/portal.js

```diff
@@ -37,6 +37,33 @@ export default class WasabeePortal {
     return wp;
   }
 
+  // create a wasabee portal from a IITC portal (leaflet marker)
+  static fromIITC(p) {
+    // we have all the details
+    if (p && p.options && p.options.data && p.options.guid) {
+      const data = p.options.data;
+      const id = p.options.guid;
+      if (data.title) {
+        return new WasabeePortal(
+          id,
+          data.title,
+          (data.latE6 / 1e6).toFixed(6).toString(),
+          (data.lngE6 / 1e6).toFixed(6).toString()
+        );
+      }
+      // do we have enough to fake it?
+      if (data.latE6) {
+        return WasabeePortal.fake(
+          (data.latE6 / 1e6).toFixed(6).toString(),
+          (data.lngE6 / 1e6).toFixed(6).toString(),
+          id
+        );
```

Comment:
Need to set a [Loading:{id}] title here.

Reply:
(no square bracket, must match("^Loading: .*")

Reply:
`Loading` is locale dependent so I delegate this to a dynamic name that add this when we need the name to be display